clippy_lints/src/doc/mod.rs

```diff
@@ -690,7 +611,7 @@ fn check_attrs(cx: &LateContext<'_>, valid_idents: &FxHashSet<String>, attrs: &[
         Some(("fake".into(), "fake".into()))
     }
 
-    if suspicious_doc_comments::check(cx, attrs) || empty_line_after::check(cx, attrs) || is_doc_hidden(attrs) {
```

Comment:
It said this check was to avoid spurious warnings somewhere, but we don't do it anymore. Is that a problem?

Reply:
I ran `cargo uibless` so I think it's ok if nothing new got picked up. To be confirmed by someone more knowledgeable though.

Reply:
It's intended for e.g. https://github.com/rust-lang/rust-clippy/issues/12917

Reply:
So what should we do here?

Reply:
It may cause some confusion but if attrs are being reworked it's something we could live without unless we want to add another early -> late storage hack